11_notify/willshen8/pkg/puppy/lostpuppy.go

```diff
@@ -0,0 +1,35 @@
+package puppy
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/render"
+)
+
+// Payload struct is the format that lost puppy server expects to see
+type Payload struct {
```

Comment:
The name of this struct could be better. "Payload" is such a generic name, and when looked at fully qualified with the package name (`puppy.Payload`), it's kind of misleading because it is not the payload of a puppy at all. I'd suggest something like `LostPuppyRequest`, because that's kind of what it is - a request to the lost puppy service.

Reply:
Cheers, I agree.